src/code/models/palette-manager.coffee

```diff
@@ -0,0 +1,76 @@
+LinkManager    = require './link-manager'
+
+# TODO: Maybe loadData goes into some other action-set
+paletteActions = Reflux.createActions(["addToPalette","loadData"])
+
+paletteStore   = Reflux.createStore
+  # NOTE: This Reflux shortcut does not work as advertised,
+  # listenables: paletteActions
```

Comment:
I did, and I was having no luck with those, see related comments in `palette-manager.coffee`